### PR TITLE
Add functionality to only add search to history once

### PR DESCRIPTION
**Pull request by DTThor**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -66,6 +67,12 @@ class App extends Component {
     this.setState({ search: '', movies: [], status: DEFAULT_STATUS, color: 'black'})
   }
 
+  clearHistory(e){
+    e.preventDefault()
+    const { history, search, status, movies } = this.state
```
Comment (arinaldi) [inline src/App.js]: Is this line needed?

File: src/components/RecentSearches.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
 import React, { Component } from 'react';
 
-const RecentSearches = ({ history, onClick }) =>
+const RecentSearches = ({ history, onClick, clearHistory }) =>
   (
-    <div className='tc dib pa2 br3 ma0 bw2 shadow-2 bg-light-blue'>
-      <h3>Last 5 Searches</h3>
+    <div className='tc pa1 br3 ma0 bw2 shadow-2 bg-light-blue'>
+      <h3 className='dib ph3'>{history.length===0 ? 'Search to start a History' : 'Search History'}</h3>
```
Comment (arinaldi) [inline src/components/RecentSearches.js]: Usually I put the 'empty state' message in place of the data (below where you are doing `history.map`), not the header.

File: src/App.js (hunk 1 above)
Comment (DTThor) [reply src/App.js]: no it doesn't seem to be. I took it out, thanks.

File: src/components/RecentSearches.js (hunk 2 above)
Comment (DTThor) [reply src/components/RecentSearches.js]: Just made this change, and I like it better. thanks. done.

**Pull request merged**